lib/vetext/response_error.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module VEText
+  # An error representing a 200 response but with error details.
```

Comment:
My understanding was VEText was a fairly new service. I'm surprised they're doing the 'error in a 200' thing. @jjulian do you think we could get them to change this?

Reply:
Is this something we can bring up in the meeting tomorrow morning?

Reply:
Based on the below chat I think we can just send the response straight through to the controllers. 
https://dsva.slack.com/archives/C01CSM3EZGT/p1615409557008500

Reply:
The very bottom of the document in that thread is where all the errors are listed out